kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/wires/WiresPanelUi.java

```diff
@@ -68,930 +68,931 @@
 
 public class WiresPanelUi extends Composite {
 
-    @UiField
```

Comment:
I believe there is some issue with the formatter as it is showing the complete change in `WiresPanelUi`. As it is very large souce, it will be difficult to find out the real changes. Could you kindly check it once?